core/block_svg.ts

```diff
@@ -241,37 +241,19 @@ export class BlockSvg
     return this.style.colourTertiary;
   }
 
+  // TODO: Before merging, is it better to just remove these or to leave them
```

Comment:
Actually we have to leave these so that the `BlockSvg` conforms to the `ISelectable` interface.

Reply:
Since you have to keep it, please add a description of the breaking change to the PR description (as I understanding, that `block.select()` and `block.unselect()` no longer fire events or update the workspace's selected value--they just update the block itself--and that devs need to call `common.select` to get the old behaviour.

Reply:
I'm gonna go through and update the PR descriptions in bulk before the release.